spinn_front_end_common/interface/front_end_common_interface_functions.py

```diff
@@ -530,17 +537,22 @@ def wait_for_execution_to_complete(
 
             processors_not_finished = self._txrx.get_core_state_count(
                 app_id, CPUState.RUNNING)
-            if processors_not_finished > 0:
-                logger.info("Simulation still not finished or failed - "
-                            "waiting a bit longer...")
-                time.sleep(0.5)
+            #if processors_not_finished > 0:
```

Comment:
Because I removed the `time.sleep` (same reason as above) and this debug message was rather clogging up the output.